core/primitives-core/src/config.rs

```diff
@@ -27,6 +27,7 @@ pub struct VMLimitConfig {
     /// Max amount of gas that can be used, excluding gas attached to promises.
     pub max_gas_burnt: Gas,
     /// Max burnt gas per view method.
+    /// TODO: remove this field because it does not affect the protocol
     pub max_gas_burnt_view: Gas,
```

Comment:
How hard is it to fix this? I'd like to prioritize this follow up, if possible, as this is a significant chunk of debt.

Reply:
ah, I didn't read the PR description properly, you *are* tackling this in the next change, thanks!